packages/app/src/templates/template.container.tsx

```diff
@@ -4,11 +4,18 @@
  */
 
 import { connect } from "react-redux"
-import { ReduxRootState } from "Renderer/store"
+import { ReduxRootState, TmpDispatch } from "Renderer/store"
 import { TemplatesList } from "App/templates/components"
+import { deleteTemplates } from "App/templates/actions"
 
 const mapStateToProps = (state: ReduxRootState) => ({
   templates: state.templates.data,
 })
 
-export default connect(mapStateToProps)(TemplatesList)
+const mapDispatchToProps = (dispatch: TmpDispatch) => ({
```

Comment:
```suggestion
const mapDispatchToProps = {
    deleteTemplates,
}
```

Reply:
Removed in [ca0ccd5](https://github.com/mudita/mudita-center/pull/752/commits/ca0ccd5977773430798c01ef71f22f8086da8e6b)